src/app/admin/AdminPage.tsx

```diff
@@ -6,13 +6,22 @@ import { Accordion, AccordionDetails, AccordionSummary } from "@mui/material";
 import { ExpandMore } from "@mui/icons-material";
 import TableSurface from "@/components/Tables/TableSurface";
 import CreateUserForm from "@/app/admin/createUser/CreateUserForm";
-import { faUsers, faUserPlus, IconDefinition } from "@fortawesome/free-solid-svg-icons";
+import {
+    faUsers,
+    faUserPlus,
+    faCity,
+    faBuildingCircleArrowRight,
+    IconDefinition,
+} from "@fortawesome/free-solid-svg-icons";
 import styled from "styled-components";
 import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
 import { UserRow } from "./page";
```

Comment:
Change to @ import (I know wasn't you but still)

Reply:
Done